torch/_inductor/ir.py

```diff
@@ -5469,6 +5435,7 @@ def __init__(
         # abi-compatible mode, where we retrieve outputs by pass each individual
         # output through the abi-compatible interface.
         self.outputs: Sequence[Any] = []
+        self.mutation_outputs: List[Buffer] = []
```

Comment:
a bit weird to have an extra attribute just in this class...

Reply:
Fair point. I've moved `mutation_outputs` into `ExternKernel` instead.